### PR TITLE
Naptol Project

### PR DESCRIPTION
**Pull request by Sagardarekar80**

### CODE REVIEW

File: src/main/java/pom/NaptolProductDetailsPage.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	}
+	
+	
+	public double getProductPrice(int index)
+	{
+		String [] a = ProductPriceList.get(index).getText().split(" ");
+		return Double.parseDouble(removeComaFromString(a[0]));	
+	}
+	
+	public void SelectProductColor(int index)
```
Comment (pratibhanhavkar) [inline src/main/java/pom/NaptolProductDetailsPage.java]: Method name should start from lowercase.

File: src/main/java/pom/NaptolQuickViewPage.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+	{
+		return ProductName.getText();
+	}
+	public double getProductPrice(int index)
+	{
+		String [] a = ProductPriceList.get(index).getText().split(" ");
+		return Double.parseDouble(removeComaFromString(a[0]));	
+	}
+	
+	public void SelectProductColor(int index)
```
Comment (pratibhanhavkar) [inline src/main/java/pom/NaptolQuickViewPage.java]: Method name should start from lowercase.

File: src/test/java/test/NaptolCartPageTest.java [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+		Thread.sleep(5000);
+		int d = naptolCartPage.getCartItemCount();
+		System.out.println(d);
+		int itemCount = naptolCartPage.getCartProductList();
+		Assert.assertEquals(itemCount, d);
+
+	}
+
+	@Test
+	public void VerifyRemovingProductFromCart() throws EncryptedDocumentException, IOException, InterruptedException {
```
Comment (renukapatil123) [inline src/test/java/test/NaptolCartPageTest.java]: Name of method should start from lower case.

File: src/test/java/test/NaptolCartPageTest.java [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+		double up=naptolCartPage.getUnitPrice();		  		
+		double sp=naptolCartPage.getShippingPrice();					
+		double tp = (up*2)+sp;			
+		Thread.sleep(5000);
+		double oa=naptolCartPage.getOrderAmount();					
+		Assert.assertEquals(oa,tp);		
+	}
+
+	@Test
+	public void VerifyTotalAmountForMultipleProductsInCart() throws EncryptedDocumentException, IOException, InterruptedException
```
Comment (renukapatil123) [inline src/test/java/test/NaptolCartPageTest.java]: method name should start from lower case

File: src/test/java/test/NaptolHomePageTest.java [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+		test = reports.createTest("verifyOnClickingShoppingCategories");
+		NaptolHomePage naptolHomePage = new NaptolHomePage(driver);
+		naptolHomePage.moveToShoppingCategories(driver);
+		Assert.assertTrue(naptolHomePage.verifyingShoppingcategoriesMenu());
+	}
+	
+	
+	//29-11-2024 changed
+	@Test
+	public void VerifyIfProdutDetailsAreCorrcetIfViewInQuickView() throws EncryptedDocumentException, IOException
```
Comment (renukapatil123) [inline src/test/java/test/NaptolHomePageTest.java]: method name should start from lower case

File: src/main/java/pom/NaptolQuickViewPage.java (hunk 2 above)
Comment (pratibhanhavkar) [reply src/main/java/pom/NaptolQuickViewPage.java:33]: ok

File: src/main/java/pom/NaptolProductDetailsPage.java (hunk 1 above)
Comment (pratibhanhavkar) [reply src/main/java/pom/NaptolProductDetailsPage.java]: ok

File: src/test/java/test/NaptolCartPageTest.java (hunk 4 above)
Comment (pratibhanhavkar) [reply src/test/java/test/NaptolCartPageTest.java:143]: ok

File: src/main/java/pom/NaptolCartPage.java [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import utility.Parameterization;
+
+public class NaptolCartPage extends BaseClass{
+
+	
+	@FindBy(xpath = "//ul[@id='cartData']//h2")private WebElement CartProductName;
```
Comment (renukapatil123) [inline src/main/java/pom/NaptolCartPage.java]: All element name should be in camel case

File: src/test/java/test/NaptolCartPageTest.java (hunk 3 above)
Comment (Sagardarekar80) [reply src/test/java/test/NaptolCartPageTest.java]: Fixed

File: src/test/java/test/NaptolCartPageTest.java (hunk 4 above)
Comment (Sagardarekar80) [reply src/test/java/test/NaptolCartPageTest.java]: Fixed

File: src/test/java/test/NaptolHomePageTest.java (hunk 5 above)
Comment (Sagardarekar80) [reply src/test/java/test/NaptolHomePageTest.java]: Fixed

File: src/main/java/pom/NaptolQuickViewPage.java (hunk 2 above)
Comment (Sagardarekar80) [reply src/main/java/pom/NaptolQuickViewPage.java]: Fixed

File: src/main/java/pom/NaptolCartPage.java (hunk 6 above)
Comment (Sagardarekar80) [reply src/main/java/pom/NaptolCartPage.java]: Fixed

**Pull request closed without merging**